asset/tests/system/test_vpcsc.py

```diff
@@ -22,67 +22,43 @@
 from google.api_core import exceptions
 from google.cloud import asset_v1
 from google.cloud.asset_v1 import enums
-
-PROJECT_INSIDE = os.environ.get("PROJECT_ID", None)
-PROJECT_OUTSIDE = os.environ.get(
-    "GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT", None
-)
-IS_INSIDE_VPCSC = os.environ.get("GOOGLE_CLOUD_TESTS_IN_VPCSC", "true")
+from test_utils.vpcsc_config import vpcsc_config
 
 
+@vpcsc_config.skip_unless_inside_vpcsc
 class TestVPCServiceControl(object):
     @staticmethod
     def _is_rejected(call):
         try:
             responses = call()
         except exceptions.PermissionDenied as e:
             return e.message == "Request is prohibited by organization's policy"
-        except:
-            pass
```

Comment:
I think we want to keep these two lines around.

If any error other than `PermissionDenied` is returned by the API (invalid request, etc.), it wasn't blocked by the VPC (which is the main reason these tests were written).